x-pack/plugins/security/public/authentication/login/components/login_form/login_form.scss

```diff
@@ -41,6 +41,14 @@
   + .secLoginCard {
     border-top: $euiBorderThin;
   }
+
+  &.secLoginCard-autoLogin {
+    border-color: transparent;
+
+    + .secLoginCard {
+      padding-top: unset;
```

Comment:
This seems to be supported well enough - since we no longer support IE - but setting this to `0` or `inherit` likely achieves the same result in a potentially more sure proof manner.